rest-api/dao/biobank_order_dao.py

```diff
@@ -124,6 +124,9 @@ def insert_with_session(self, session, obj):
     inserted_obj = super(BiobankOrderDao, self).insert_with_session(session, obj)
     ParticipantDao().add_missing_hpo_from_site(
         session, inserted_obj.participantId, inserted_obj.collectedSiteId)
+    self._update_history(session, obj)
```

Comment:
These methods get called in three different places, and always get called together. Consider combining them? (Or having a helper method that calls all three which gets invoked in the three places?)

Reply:
done